README.md

```diff
@@ -8,15 +8,24 @@ Coming soon...
 
 ## Usage
 
-Coming soon...
+```js
+const sdk = require('eventbrite')({token: 'OATH_TOKEN_HERE'});
+
+// See: https://www.eventbrite.com/developer/v3/endpoints/users/#ebapi-get-users-id
+sdk.request('/users/me').then(res => {
```

Comment:
so I think we shouldn't be encouraging people to use .request method directly and instead provide them wrappers for particular objects like `getUser` or `getEvents` so that when the endpoint changes the api kit will mask the change

Reply:
I agree! I think it serves a purpose though and I think we should definitely make it available to our consumers. Maybe when the rest of our main apis get fleshed out we can move those to the forefront of our documentation, and note that they are slightly more stable and preferred.

Reply:
I agree. When we have the wrappers this example will get replaced. But the nice thing is that someone (like core) could start using it now with just `.request()`

Reply:
I imagine a use-case for `.request()` even when we do have the wrappers is if a new API is added and we haven't gotten around to building the wrapper for it. I believe it should always be made available in the SDK, just not in the forefront of docs

Reply:
yup, exactly. Basically it's `you're on your own` method because we can change/deprecate the endpoints, whereas with the wrappers you're guaranteed to get some type of response back